toolchains/install_bootstrap/musl/setup.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
```

Comment:
The only things I changed in this file are the chmods at the bottom, changing the directories at the top, and the copyright statement